src/components/10-atoms/radio/index.scss

```diff
@@ -166,18 +166,19 @@ axa-radio {
       height: auto;
 
       .a-radio__wrapper {
-          padding: 26px 35px 0 35px;
+          padding: 10px 41px 0 41px;
           flex-direction: column;
           justify-content: center;
 
         svg {
-          width: 60px;
+          width: 48px;
+          height: 48px;
           fill: currentColor;
         }
 
         .a-radio__content {
-          padding-bottom: 15px;
-          padding-top: inherit;
+          padding-bottom: 10px;
+          padding-top: 10px;
```

Comment:
Not sure why it was inherit, maybe there was a reason. But if you let it as "inherit" it will still be 10px.

But just a comment, I have no opinion on this.

Reply:
Same like Raphi. If you exactly know why we have `inherit` here and it is not necessary, you can remove it. Otherwise not.

Reply:
Inherit just get its value from its parent element. `.a-radio__content` has a parent `.a-radio__wrapper`. The parents value for `padding-top` is also `10px`. In this case It's either way
10px. But because the padding of the wrapper and the content are independent, this is the only way to do it in my opinion.

Reply:
Commit where it was implemented. 
https://github.com/axa-ch/patterns-library/commit/606ad523db71b55817807fd01fa5a6e0562f32e7